.circleci/config.yml

```
@@ -54,3 +63,65 @@ jobs:
       - store_artifacts: *artifact_env_min
       - save_cache: *save-node-modules-cache
       - save_cache: *save-yarn-cache
+  test262:
+    working_directory: ~/babel
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - restore-cache: *restore-yarn-cache
+      - restore-cache: *restore-node-modules-cache
+      - run:
+          name: Build Babel
+          command: BABEL_ENV=test make bootstrap
+      - run:
+          name: Link Babel
+          command: |
+            cd packages
+            for package in */; do
+              cd $package
+              yarn link
+              cd ..
+            done
+      - run:
+          name: Setup Test Runner
+          command: |
+            git clone --recurse-submodules https://github.com/babel/babel-test262-runner
+            cd babel-test262-runner
+            yarn
+            yarn add tap-mocha-reporter --dev
+            curl -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 > jq
+            chmod +x ./jq
+            for package in ../packages/*/package.json; do
+              yarn link $(./jq -j ".name" $package)
+            done
+            node lib/download-node
+      - run:
+          name: Run Test262
+          command: node lib/run-tests I_AM_SURE | tee ~/test262.tap
+          <<: *test262_workdir
+      - store_artifacts: *artifact_test262_tap
+      - run:
+          name: Convert Tap output to Xunit
+          command: |
+            mkdir -p ~/test-results/test262
+            cat ~/test262.tap | $(npm bin)/tap-mocha-reporter xunit | tee ~/test-results/test262/results.xml
+          <<: *test262_workdir
+      - store_test_results: *artifact_test262_xunit
+
+
+workflows:
+  version: 2
+  test:
+    jobs:
+      - test
+  nightly:
+      triggers:
+        - schedule:
+            cron: "0 0 * * *"
```

According to https://api.github.com/repos/babel/babel/stats/punch_card, it's better to to this at

```suggestion
            cron: "0 7 * * *"
```

(is I'm understanding the "cron" format correctly: it should be 7 am UTC)

| Time | Commits |
|------|---------|
| 0 | 539 |
| 1 | 427 |
| 2 | 298 |
| 3 | 213 |
| 4 | 174 |
| 5 | 136 |
| 6 | 201 |
| 7 | 164 |
| 8 | 309 |
| 9 | 399 |
| 10 | 552 |
| 11 | 686 |
| 12 | 546 |
| 13 | 563 |
| 14 | 599 |
| 15 | 634 |
| 16 | 698 |
| 17 | 628 |
| 18 | 634 |
| 19 | 507 |
| 20 | 475 |
| 21 | 566 |
| 22 | 659 |
| 23 | 704 |

---

confirmed that it's the right syntax